controlplane/kubeadm/internal/workload_cluster_conditions.go

```diff
@@ -154,6 +184,9 @@ func (w *Workload) updateManagedEtcdConditions(ctx context.Context, controlPlane
 		if err != nil {
 			// Note. even if we fail reading the member list from one node/etcd members we do not set EtcdMembersAgreeOnMemberList and EtcdMembersAgreeOnClusterID to false
 			// (those info are computed on what we can collect during inspection, so we can reason about availability even if there is a certain degree of problems in the cluster).
+
+			// While scaling up/down or rolling out new CP machines this error might happen.
+			retryableError = true
```

Comment:
I don't remember why it's okay to simply continue here, but that's independent of this PR